mapbox_gl_platform_interface/lib/src/method_channel_mapbox_gl.dart

```diff
@@ -123,19 +122,19 @@ class MethodChannelMapboxGl extends MapboxGlPlatform {
   Widget buildView(
       Map<String, dynamic> creationParams,
       Function onPlatformViewCreated,
-      Set<Factory<OneSequenceGestureRecognizer>> gestureRecognizers) {
+      Set<Factory<OneSequenceGestureRecognizer>>? gestureRecognizers) {
     if (defaultTargetPlatform == TargetPlatform.android) {
       return AndroidView(
         viewType: 'plugins.flutter.io/mapbox_gl',
-        onPlatformViewCreated: onPlatformViewCreated,
+        onPlatformViewCreated: onPlatformViewCreated as void Function(int)?,
```

Comment:
Should the type `OnPlatformViewCreatedCallback` be reused here?

The cast is unfortunate but I'm assuming that the type is not compatible across the mobile and web platforms?

Reply:
Good catch, thanks!